katello/hooks/boot/13-hiera.rb

```diff
@@ -0,0 +1,16 @@
+# TODO automatically get the tuning sizes
+TUNING_SIZES = ['small', 'medium', 'large', 'extra-large', 'extra-extra-large']
```

Comment:
I lean towards `extra-extra-large` as that follows on `extra-large`.

Reply:
I mentioned `2-extra-large` because it felt like `extra` was redundant in `extra-extra-large` and also [satellite-support](https://github.com/RedHatSatellite/satellite-support/blob/master/tuning-profiles/custom-hiera-2ex-large-256G.yaml) uses `2ex` convention so users may be already familiar with it.

I am ok with either though.